### PR TITLE
Dat 4062

### PR DESCRIPTION
**Pull request by idradm**
## Links

* http://wikia-inc.atlassian.net/browse/DAT-4062

## Description

ABTest changes

## Reviewers

@Wikia/west-wing

### CODE REVIEW

File: front/main/app/styles/component/_site-head.scss [hunk 1]
```diff
@@ -34,6 +34,19 @@ $site-head-height: 50; //unitless
 			height: 3.125rem;
 			width: 5rem;
 		}
+
+		// temporary change for nav entry points AB test - https://wikia-inc.atlassian.net/browse/DAT-4052
+		// TODO: cleanup as a part of https://wikia-inc.atlassian.net/browse/DAT-4064
+		.dropdown:after {
+			background-image: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNyIgaGVpZ2h0PSI0IiB2aWV3Qm94PSIwIDAgNyA0IiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPgogIDxwYXRoIGQ9Ik0wIDBoNy4wMzNMMy41MTYgMy41MjYgMCAweiIgZmlsbD0iIzk5OSIvPgo8L3N2Zz4K');
+			content: " ";
```
Comment (RafLeszczynski) [inline front/main/app/styles/component/_site-head.scss]: content can be empty string

File: front/main/app/templates/components/application-wrapper.hbs [hunk 2]
```diff
@@ -46,12 +50,14 @@
 		}}
 	{{/if}}
 
-	{{#if enableShareHeader}}
-		{{share-header
-			smartBannerVisible=smartBannerVisible
-			title=currentTitle
-			language=language
-		}}
+	{{#if navABTestEnableShare}}
```
Comment (RafLeszczynski) [inline front/main/app/templates/components/application-wrapper.hbs]: please add comment about code change for the experiment

**Comment by RafLeszczynski** [comment]
LG for experiment purpose  :-)

**Comment by SebastianMarzjan** [comment]
LG(for an experiment)

**Pull request merged**